### ISSUE TITLE
0.2.0: Exception handling needs to be more complete

### ISSUE THREAD

**Issue by GeorgeBailey** [jsdom/jsdom#244]
Right now, for some reason, exceptions are silently dropped when thrown from within the callback of jqueryify or when thrown internally.

Test code

jsdom.jQueryify(window,'jquery.js',function () {throw new Error()})

If an error is thrown and nothing happens, it is very difficult to debug. Especially when it is thrown internally, for example garbage after &lt;/html> or some incorrect data in the &lt;head>&lt;/head>, there is bound to be some fringe that will through an error. And there is bound to be some programmer mistake in which case you should handle my test code above.

Basically, either a callback needs to happen, or an error should be thrown, or an error event emitted.

<i>Edited: Sorry, I did not notice everything after the word "after" was stripped out because of the &lt;</i>
